yt/frontends/gadget/io.py

```diff
@@ -14,7 +14,12 @@
 
 class IOHandlerGadgetHDF5(IOHandlerSPH):
     _dataset_type = "gadget_hdf5"
-    _vector_fields = ("Coordinates", "Velocity", "Velocities", "MagneticField")
+    _vector_fields = {
```

Comment:
I'm going to make this comment here, on a PR I originally authored, just to note that this is one place that a *proper* DSL, or even using the DSL we have *already defined* for Gadget-like fields would be extremely useful and helpful.

Reply:
agreed!